plugins/index.js

```diff
@@ -196,9 +190,8 @@ export async function loadPluginsIfNecessary() {
 
     // Remove old plugins
     Object.keys(oldManifests).forEach((id) => {
-        if (!newManifests.hasOwnProperty(id)) {
+        if (!Object.prototype.hasOwnProperty.call(newManifestsMap, id)) {
```

Comment:
I think we could just check for the existance of `newManifests[id]` here. I don't think we should be adding a dependency on `lodash` when there's a built-in way to do this.

Reply:
Fair point. It is generally not a good idea to apply a boolean operator on a falsy value `undefined`, `null`, `''` etc., which is why I had recommended `.has`. But you could use `if (!Boolean(newManifestMap[id]))` instead.

https://stackoverflow.com/a/4735055/5551762

Reply:
I do think we could do more of that, but it's not something we've really enforced in the past. Falsiness is something I know I kind of accepted when learning JS in the first place 😛.

In this case, I'd prefer to use something built-in, but then I'm 0/5 whether that's using `hasOwnProperty` or just checking `!newManifestMap[id]`/`!Boolean(newManifestMap[id])`.

Reply:
@ali-farooq0 are you sure Boolean casting is needed here?  I can't come up with any reason for use of it and even `no-extra-boolean-cast` eslint rule says it is redundant.

Reply:
@Qovaros 

Fair enough. Yeah I was just trying to avoid implicit boolean casting, with `Boolean()` it's obvious we're checking for a bool rather than converting an `!undefined` -> `true`. But I suppose we have no choice but to obey eslint :P